doc/refman/RefMan-ext.tex

```diff
@@ -279,15 +279,64 @@ \subsection{Primitive Projections}
 \label{prim-proj}
 
 The option {\tt Set Primitive Projections} turns on the use of primitive
```

Comment:
A lot of index commands are missing from this section, for the `Primitive Projections Commands`, and for subsequent commands that get documented here.

Reply:
There's the three optindex for the three settable options, what more do you want?

Reply:
Hum, indeed. Sorry for the noise.